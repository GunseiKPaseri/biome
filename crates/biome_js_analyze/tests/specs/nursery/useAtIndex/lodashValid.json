[
  "new _.last(array)",
  "_.last(array, 2)",
  "_.last(...array)"
]
