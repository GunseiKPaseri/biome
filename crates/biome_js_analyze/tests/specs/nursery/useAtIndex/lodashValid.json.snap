---
source: crates/biome_js_analyze/tests/spec_tests.rs
assertion_line: 86
expression: lodashValid.json
---
# Input
```cjs
new _.last(array)
```

# Input
```cjs
_.last(array, 2)
```

# Input
```cjs
_.last(...array)
```
